packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -162,12 +162,13 @@ function makeOperationGenerator(optionsParam?: OperationGenerationConfig): Gener
 
     function inclusiveRange(state: ClientOpState): RangeSpec {
         const start = position(state);
-        const end = state.random.integer(start, state.sharedString.getLength() - 1);
+        const end = state.random.integer(start, Math.max(start, state.sharedString.getLength() - 1));
```

Comment:
Ditto: Should this range include the position after the last character of the SharedString?

Reply:
nope--see other comment.

Btw, I'm curious if this one actually caused a failure (I can see how the other one definitely does). There are rejection criteria on which ops to generate, and I don't think we should ever be calling `inclusiveRange` with an empty string.

Reply:
Probably not.  I hit one failure, and then started grepping the codebase for other instances that could be impacted by my behavior change.